man/systemd-homed-sync-bulk.service.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+
+<!--
+  SPDX-License-Identifier: LGPL-2.1-or-later
+
+  Copyright © 2023 GNOME Foundation Inc.
+        Original Author: Adrian Vovk
+-->
+
+<refentry id="systemd-homed-sync-bulk.service" conditional='ENABLE_HOMED'
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>systemd-homed-sync-bulk.service</title>
```

Comment:
hmm, why make this a service of its own? I would just put this inside of homework.c i.e. as part of the reconciliation of the user record in the home dir and on the host. I mean, we are not talking about copying gigabytes here, but just a couple of 10M or so.

Reply:
We want the data to be copied over continuously, on any change. Not just during activation of the home dir. Hence it's a separate service that gets triggered via a path

Reply:
do we really want that? i hate such async interfaces, since you never know when its complete. why not just make the relevant UI that drop stuff there call some bus call that syncs this down. the UI woudl then get a report when this is done. 

i.e. add a new homed bus call SynchronizeHomeBulk()?

Reply:
Makes sense to me, but which direction is that syncing? Or is it syncing both ways?